src/containers/AdminCampaignEdit/components/StartCampaignButton.tsx

```diff
@@ -38,11 +47,32 @@ export const StartCampaignButton: React.FC<StartCampaignButtonProps> = (
 
   const tooltipText = requiresApproval ? "Superadmin approval required" : "";
 
+  const campaignInteractionSteps =
+    campaignStepsData?.campaign?.interactionSteps ?? [];
+
+  const campaignHasSurveyResponses = campaignInteractionSteps.length > 1;
```

Comment:
I couldn't tell from the ticket if we want/need to be more thorough than checking that a campaign has more than one interaction step to determine if the admin should be shown the warning dialog. Happy to add more logic + fields to the graphql query above, if there's additional info about the interaction steps needed.

Reply:
this is fine imo!